lib/Parser/Parse.h

```diff
@@ -380,7 +377,6 @@ class Parser
     ParseNodePtr CreateIntNodeWithScanner(int32 lw);
     ParseNodePtr CreateProgNodeWithScanner(bool isModuleSource);
 
-    static void InitNode(OpCode nop,ParseNodePtr pnode);
     static void InitBlockNode(ParseNodePtr pnode, int blockId, PnodeBlockType blockType);
 
```

Comment:
Should I go through the exercise of replacing this static method with an instance Init as well?

Reply:
If you're feeling motivated.